crates/biome_css_formatter/src/css/statements/position_try_at_rule.rs

```diff
@@ -0,0 +1,10 @@
+use crate::prelude::*;
+use biome_css_syntax::CssPositionTryAtRule;
+use biome_rowan::AstNode;
+#[derive(Debug, Clone, Default)]
+pub(crate) struct FormatCssPositionTryAtRule;
+impl FormatNodeRule<CssPositionTryAtRule> for FormatCssPositionTryAtRule {
+    fn fmt_fields(&self, node: &CssPositionTryAtRule, f: &mut CssFormatter) -> FormatResult<()> {
+        format_verbatim_node(node.syntax()).fmt(f)
```

Comment:
We should format these new nodes. You can take inspiration from how some of the other at-rules are formatted.

Reply:
Ah right i missed that, i've just added it!